tensorflow/core/kernels/data/experimental/map_and_batch_dataset_op_test.cc

```diff
@@ -316,17 +316,17 @@ DATASET_OUTPUT_SHAPES_TEST_P(MapAndBatchDatasetOpTest, MapAndBatchDatasetParams,
 std::vector<CardinalityTestCase<MapAndBatchDatasetParams>>
 CardinalityTestCases() {
   return {{/*dataset_params=*/MapAndBatchDatasetParams1(),
-           /*expected_cardinality=*/2},
+           /*expected_cardinality=*/-2},
```

Comment:
You can use the [kUnknownCardinality](https://github.com/tensorflow/tensorflow/blob/b3925cc9b62775904734eb789da98bd3d0a9dd0b/tensorflow/core/kernels/data/interleave_dataset_op_test.cc#L396) constant for these

Reply:
fixed.